go/vt/sqlparser/ast.go

```diff
@@ -725,6 +725,12 @@ func (node *TableName) Equal(t *TableName) bool {
 	return node.Name == t.Name && node.Qualifier == t.Qualifier
 }
 
+// Lowered returns a new TableName where the Name and Qualifier have been converted
+// to lower case
+func (node *TableName) Lowered() *TableName {
+	return &TableName{Name: node.Name.Lowered(), Qualifier: node.Qualifier.Lowered()}
```

Comment:
Looks like you're using this for lower-casing view names. If so, `Qualifier` should not be lowered. So, you may have to also rename this function to `ToViewName`.